core/src/database/mledb/PlayerStatsCore.model.ts

```diff
@@ -56,12 +56,27 @@ export class MLE_PlayerStatsCore {
     @Column("integer", {name: "assists"})
     assists: number;
 
+    @Column("integer", {name: "goals_against"})
+    goals_against: number;
+
+    @Column("integer", {name: "shots_against"})
+    shots_against: number;
+
```

Comment:
Need to be nullable

Reply:
I don't see why these need to be nullable but the other core stats don't.

Reply:
It doesn't exist in every column that already exists, so we either need a default or a nullable, nullable would make more sense to me because we don't have the legacy data

Reply:
Done.